java/src/test/java/com/jongallant/azure/identity/extensions/DefaultAzureServiceBusCredentialTest.java

```diff
@@ -0,0 +1,79 @@
+package com.jongallant.azure.identity.extensions;
+
+import com.azure.identity.DefaultAzureCredential;
+import com.microsoft.azure.servicebus.ClientSettings;
+import com.microsoft.azure.servicebus.ExceptionPhase;
+import com.microsoft.azure.servicebus.IMessage;
+import com.microsoft.azure.servicebus.IMessageHandler;
+import com.microsoft.azure.servicebus.Message;
+import com.microsoft.azure.servicebus.ReceiveMode;
+import com.microsoft.azure.servicebus.SubscriptionClient;
+import com.microsoft.azure.servicebus.TopicClient;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executors;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit test for {@link DefaultAzureServiceBusCredential}.
+ */
+public class DefaultAzureServiceBusCredentialTest {
+
+    private static final String AZURE_SERVICE_BUS_ENDPOINT = "AZURE_SERVICE_BUS_ENDPOINT";
```

Comment:
FYI - You are also going to need to set the other env vars to test envcred, but you don't have to do that incode, just in the .env and the dotenv package will pick it up, add it to env vars, and then envcred will pick them up.

Reply:
Tested with all environment variables locally and everything worked fine (not checking in the service bus names, client ids etc)